platforms/common/src/main/resources/assets/dynamic_fps/lang/de_at.json

```diff
@@ -33,10 +69,20 @@
     "config.dynamic_fps.run_garbage_collector": "GC auslösen",
     "config.dynamic_fps.run_garbage_collector_tooltip": "Ungenutzen Arbeitsspeicher frei machen, wenn dieser Status aktiviert wird",
 
+    "config.dynamic_fps.download_natives": "Bibliotheken Herunterladen",
+    "config.dynamic_fps.download_natives_description_0": "Nutzung der Akku-Features erfordert eine weitere Bibliothek.",
+    "config.dynamic_fps.download_natives_description_1": "Stelle ein, ob die Mod diese für dich herunterladen darf.",
+
     "key.dynamic_fps.toggle_forced": "Unfokussierten Modus forcieren (Toggle)",
     "key.dynamic_fps.toggle_disabled": "Dynamic FPS deaktivieren (Toggle)",
     "gui.dynamic_fps.hud.reducing": "Dynamic FPS: tiefe FPS forciert",
     "gui.dynamic_fps.hud.disabled": "Dynamic FPS deaktiviert",
 
-    "modmenu.descriptionTranslation.dynamic_fps": "Ändert die maximale Bildrate, damit Minecraft weniger Ressourcen im Hintergrund braucht."
+    "toast.dynamic_fps.battery_charging": "Akku Lädt!",
+    "toast.dynamic_fps.battery_draining": "Akku Entlädt!",
+    "toast.dynamic_fps.battery_critical": "Akkustand Niedrig!",
+
+    "toast.dynamic_fps.battery_charge": "Akkustand %d%%",
+
+    "modmenu.descriptionTranslation.dynamic_fps": "Reduziere die Ressourcennutzung, während Minecraft im Hintergrund oder inaktiv ist."
```

Comment:
`Reduziert` would more closely match the previous wording, and I think it fits better.